### PR TITLE
fix: app crash on clicking send while scanning

### PR DESCRIPTION
**Pull request by anhanh11001**
Detail:
Only enable startScan function if isScanning flag is false

Fixes: #129

Screenshots for the change:

### CODE REVIEW

**Comment by anhanh11001** [comment]
@yashk2000 @yatri1609 @naman653 Please peer review. By the way, I'm starting to this project so please if there are any tips on working with this, I would love to here. Thanks guys

**Comment by yashk2000** [comment]
Invalid pull request and issue. The app is supposed to send a default bitmap to the device if the edittext is empty.

**Comment by anhanh11001** [comment]
@yashk2000 I think you misunderstand the PR and the issue. It's not about empty edittext, it's about clicking send twice, which invoke the function startLeScan from ScanHelper twice and crash the device

File: app/src/main/java/com/nilhcem/blenamebadge/device/bluetooth/ScanHelper.kt [hunk 1]
```diff
@@ -43,21 +43,23 @@ class ScanHelper {
     }
 
     fun startLeScan(onDeviceFoundCallback: ((BluetoothDevice?) -> Unit)) {
-        this.onDeviceFoundCallback = onDeviceFoundCallback
-        isScanning = true
+        if (!isScanning) {
```
Comment (iamareebjamal) [inline app/src/main/java/com/nilhcem/blenamebadge/device/bluetooth/ScanHelper.kt]: Use early return

Comment (anhanh11001) [reply app/src/main/java/com/nilhcem/blenamebadge/device/bluetooth/ScanHelper.kt]: @iamareebjamal I have fixed that

**Pull request merged**